packages/repo-tools/src/commands/openapi/client/generate.ts

```diff
@@ -36,10 +36,9 @@ async function generate(spec: string, outputDirectory: string) {
   );
 
   await exec(
-    // The actual main.js file for the binary isn't executable but yarn does _something_ to make it executable.
-    // TODO (sennyeya@): Make this use the actual binary
-    `yarn openapi-generator-cli`,
+    'node',
     [
+      resolvePackagePath('@openapitools/openapi-generator-cli', 'main.js'),
```

Comment:
I don't think I quite got why this change was made. Seems very unfortunate and needs us to know about the internals of how the package is developed.

There seems to be [a couple of other methods of pointing out the config file](https://github.com/OpenAPITools/openapi-generator-cli/blob/master/apps/generator-cli/src/app/services/config.service.ts#L10C28-L10C28) that we might try to use instead of having to change how the thing executes and what its working dir is.